src/hooks/usePlayer.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+import { randomTetromino } from '../utils/tetrominos';
+
+const usePlayer = () => {
+  const [player, setPlayer] = useState({
+    pos: { x: 0, y: 0 },
+    tetromino: randomTetromino().shape,
+    collided: false,
+  });
+  return player;
```

Comment:
for now, it's ok but when you need to return more than one value you could return an array containing your values and when you called them you can make destructure like this const [ player ] = usePlayer()